sections/06-vibrant-bodies.md

```diff
@@ -0,0 +1,201 @@
+
+I argue that, as a result of this failure to confront the material possibilities of software,  "free software" and "open source", as methods and as a political movements, remain largely technocratic constructions. Creative control remains centralized in software's programming language source code, where only a limited community--those who have access to the specialized knowledge of computer programming--are able to benefit from its purported "liberty" and "freedom." Via my software development practice, manifest here in _Flocking_, _Aconite_, _In Passing_, _Tofino_, and _Font Màgica_, I aim to extend the values and methods of open community practice into the material of software.
+
+## Material Change
+
+Wendy Chun: "Richard Stallman, in his critique of nonfree software, has argued that an executable program 'is a mysterious bunch of numbers. What it does is secret.' Against this magical execution, source code supposedly provides an understanding and a freedom--the ability to map and know the workings of the machine, but, again, only through a magical erasure of the gap between source and execution, an erasure of execution itself" (51).
+
+This Cartesian dualism that Chun refers to, source/execution, haunts computation to its core. It is implicated in a related binary that Chun overlooks, the separation of _code_ from _data_. These dualisms crucially influence the way in which software's form and potential is commonly understood today, and account for how its materiality is often overlooked or fails to actualize at all. Conventional source code, before it can be used, must be transformed into something different: a binary executable created by a _compiler_. This transformation is unidirectional; as Chun suggests, it is an act of erasure that enacts the practical gap between source code, which is understood as fundamentally changeable and authorable--the "source" of creative power--and execution, which freezes the creative act and transforms it into an object and product. While code is apparently iterative and endlessly modifiable (by those with the skills and access, anyway), executables are not. One cannot unravel a piece of software and turn it into something else.
```

Comment:
"Curse you, Cartesian dualism! You've struck again!" said every heuristic practitioner from the 17th century onwards.